docs/faqs.md

```diff
@@ -319,11 +319,17 @@ features that will be broken unless you also upgrade Conan client (and sometimes
 need to modify your project if the recipe changes the binaries, flags,... it provides).
 
 To isolate from this changes there are different strategies you can follow:
+
+The minimum solution involves small changes to your Conan client configuration by
+
+ * **Pin the version of every reference you consume in your project** using recipe revisions
+   and lockfiles.
+   
+For larger projects and teams it is recommended to add some infrastructure to ensure stability by
+
```

Comment:
It's quite vague for many people I think. A link to documentation with a concrete example would help.

Reply:
Theres a link below to a page with more documentation just outside of the diff.

If you didn't see that, I can move it up to make it more obvious

Reply:
links should be inline, the word `revisions` should point to the revisions doc, and the word `lockfile` should point to the lockfiles doc. for newbies, these words probably say nothing itself.